src/json.hpp.re2c

```diff
@@ -4618,14 +4837,16 @@ class basic_json
     */
     friend bool operator<(const value_t lhs, const value_t rhs)
     {
-        static constexpr std::array<uint8_t, 7> order = {{
+        static constexpr std::array<uint8_t, 9> order = {{
```

Comment:
See the comment regarding the order inside the enum above. I'm not sure if still 9 values are required here.